vcell-client/src/main/java/cbit/vcell/client/BNGWindowManager.java

```diff
@@ -148,7 +149,7 @@ public void importSbml(String bngSbmlStr) {
 
 			@Override
 			public void sendMessage(Priority p, ErrorType et, String message)
-					throws Exception {
+					throws VCLoggerException {
 				System.err.println("LOGGER: msgLevel="+p+", msgType="+et+", "+message);
 				if (p == VCLogger.Priority.HighPriority) {
 					throw new RuntimeException("Import failed : " + message);

```

Comment:
this is the whole point of the Exception.

if (p == VCLogger.Priority.HighPriority) {
   throw new VCLoggerException(...);
}

if we are going to only use runtime exceptions ... (if that is the only thing being thrown right now within sendMessage() then we can remove the throws clause completely.